Parsers/ASimNetworkSession/Tests/Cisco_ISE_NetworkSession_DataTest.csv

```diff
@@ -0,0 +1,9 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (100.0%) for field [EventProduct] of type [Enumerated]: [""ISE""] (Schema:NetworkSession)"
+"(1) Warning: Empty value in 10 records (100.0%)  in mandatory field [Dvc] (Schema:NetworkSession)"
+"(2) Info: Empty value in 10 records (100.0%)  in optional field [DstMacAddr] (Schema:NetworkSession)"
+"(2) Info: Empty value in 10 records (100.0%)  in optional field [DstUsername] (Schema:NetworkSession)"
+"(2) Info: Empty value in 10 records (100.0%)  in optional field [EventOriginalResultDetails] (Schema:NetworkSession)"

```

Comment:
This value was coming from the lookup table. Why is this empty?

Reply:
added new test results